docs/source/development/documentation/overview.md

```diff
@@ -0,0 +1,125 @@
+    jupytext_version: 1.13.8
+---
+
+# Documentation system overview
+
+The Virtual Rainforest project is documented using the
+[`sphinx`](https://www.sphinx-doc.org/en/master/) document generation system. This
+includes documentation to:
+
+* present the scientific background underpinning the `virtual_rainforest` package,
+* provide tutorials in using the package,
+* demonstrate how to use the package components in more detail, and
+* technical details of the application program interface (API) of the underlying code.
+
+The project makes use of the following technologies within `sphinx` to structure the
+documentation content and then render that content as the [project
+website](https://virtual-rainforest.readthedocs.io/).
+
+## Sphinx build process
+
+The documentation sources for `virtual_rainforest` are stored in the `docs/source`
+directory along with the `sphinx` configuration file `conf.py`. As noted below, the
+`sphinx` build process will need to run and build Jupyter notebooks. This requires the
+extra setup step shown below and [explained
+here](jupyter_notebooks.md#jupyter-kernel-setup).
+
+```zsh
+poetry run python -m ipykernel install --user --name=vr_python3
+```
+
+The HTML documentation can be built from `docs` folder from the command line using the
+commands below and will be built in  the `docs/build/html` directory. You can open
+`docs/build/html/index.html` in a browser to see the documentation.
+
+```sh
+cd docs/
+# Optionally, to rebuild from scratch
+make clean
+# To build the HTML pages
+make html
+```
+
+The `make html` command will only build pages for files that have changed recently, and
+it can sometimes be necessary to use `make clean` to remove all of the existing built
+pages in order to rebuild  the documentation from scratch.
+
+## MyST Markdown
+
+All of the documentation apart from code docstrings is written using [MyST
+Markdown](https://myst-parser.readthedocs.io/), which provides an extended set of
+Markdown features. MyST also provides a parser for MyST Markdown content (`myst-parser`)
+that allows pages written in MyST to be rendered in `sphinx`. It is a simple replacement
+for the [RST format](https://docutils.sourceforge.io/rst.html).
+
+## Jupyter Notebooks
+
+In addition to static content, both 'tutorial' and 'how to' pages can contain actual
+Python code demonstrating how to use the components of the `virtual_rainforest` package.
+These pages are written using [Jupyter](https://jupyter.org/) notebooks. These notebooks
+can be worked on interactively by developers using [`jupyter`](https://jupyter.org/) and
+can also be run by `sphinx`, using the [`myst-nb`
+extension](https://myst-nb.readthedocs.io/), to automatically run the code in the
+notebooks and then convert the content into web pages. We use the MyST Markdown format
+for writing Jupyter notebooks, making use of the [`jupytext`
+extension](https://jupytext.readthedocs.io/) to Jupyter.
+  
+```{admonition} More information
```

Comment:
The "admonition" seems to imply that these blocks are for things you shouldn't be doing.

Is there a different block type for extra info? Or does the formatting look fine as is?

Reply:
I completely agree that using `admonition` seems loaded, but that seems to be the generic name for this kind of box. There _are_ different kinds but they have fixed box titles and the generic `admonition` is the only one that allows you to set the title: https://jupyterbook.org/en/stable/reference/cheatsheet.html#admonitions

Weirdly, there doesn't seem to be an `info` or `information` option.

Reply:
Fair enough, just thought I'd check!